### ISSUE TITLE
Make import a bit more resilient 

### ISSUE THREAD

**Issue by filipposc5** [ministryofjustice/postcodeinfo#143]
It may be useful to have exit code checking for individual commands within download_and_import_all.sh, and exception handling in the code if certain files fail to import and so on. The rationale is that it might be useful to have a way to be notified if an import failed. 

In addition it might be worth incorporating some logging around the import.

**Comment by cliffxuan** [comment]
this ticket is related to this one https://github.com/ministryofjustice/postcodeinfo/issues/145

**Comment by filipposc5** [comment]
Yes as long as both aspects are taken into consideration. Ie you might detect that everything is running fine via exit codes but would be good to have a periodic progress info as well say every 25% or something like that because now it can take hours without any indication if it is working.

**Comment by cliffxuan** [comment]
yes, will try to take both into consideration. both are valid points.

have you used luigi from spotify before. https://github.com/spotify/luigi i've used it in multiple occasions, and think it might fit this quite well.
